src/collections/pages/en/projects.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Projects
+permalink: en/projects/
```

Comment:
You shouldn't need to specify this if `generatePermalink` is properly set up, let's discuss.

Reply:
https://github.com/fluid-project/eleventy-plugin-fluid/blob/dev/fixtures/posts/posts.11tydata.js